.github/ionic-issue-bot.yml

```diff
@@ -24,12 +24,26 @@ closeAndLock:
   lock: true
   dryRun: false
 
-stale:
+noReply:
   days: 30
   maxIssuesPerRun: 100
+  includePullRequests: false
+  label: Awaiting Reply
+  responseLabel: Reply Received
+  close: false
+  lock: false
+  message: >
+    Hey there! Just a friendly reminder that the Stencil team would like some additional
+    input on this issue. Thanks!
```

Comment:
One VERY pedantic thing: "Thanks" in text can be read as passive aggressive more easily than "Thank you!" - could we reword this to "Thank you!"?